src/components/Button.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react-native/no-unused-styles */
+import { ColorTheme } from "@/theme/colors";
+import { Space } from "@/theme/layouts";
```

Comment:
I like this approach to theming more than the dependency injection everyone else uses but I think the imports are going to get overwhelming though. What are your thoughts that we just make one `theme.ts` file that hosts all of the constants?